### DISCUSSION TITLE
Compiling and running compiled version with node . opens up a new VSCode Window

### DISCUSSION THREAD

**Discussion by sevetseh28** [coder/code-server#6994] [Q&A] [answered]
Hi, im running the project within a devcontainer.
I successfully compiled it, but when I do:
`node .` in the `release` folder as stated by the docs, I get:

```
vscode ➜ /workspaces/code-server/release (main) $ node . --auth none
[2024-09-10T14:14:31.570Z] error Please specify at least one file or folder
```

So I run:
`node . .`

And right after that it opens a new VSCode Window (of my local VSCode IDE) instead, and the server doesn't start.
Any ideas? 

Thanks!

**Comment by code-asher** [comment, marked as answer]
Is this inside VS Code's integrated terminal?  When inside the integrated terminal code-server always tries to open a file in the existing code-server instance.  It does this by looking for the `VSCODE_IPC_HOOK_CLI` environment variable, so one way to get this to work is to unset it:


```
VSCODE_IPC_HOOK_CLI= node . --auth none
```

**Comment by sevetseh28** [comment]
Thank you so much for the insight! that worked 👌
